leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -58,12 +66,35 @@ private void initialize() {
 
     @Override
     public void writeRound(long seq, V round) {
-        execute(dao -> dao.writeRound(namespace, useCase, seq, round.persistToBytes()));
+        try {
+            execute(dao -> dao.writeRound(namespace, useCase, seq, round.persistToBytes()));
+        } catch (Exception ex) {
+            log.error(
+                    "Failed to write round",
+                    SafeArg.of("namespace", namespace),
+                    SafeArg.of("useCase", useCase),
+                    SafeArg.of("seq", seq),
+                    UnsafeArg.of("value", round),
+                    ex);
+            throw ex;
+        }
     }
 
     @Override
     public void writeBatchOfRounds(Iterable<PaxosRound<V>> rounds) {
-        execute(dao -> dao.writeBatchOfRounds(namespace, useCase, rounds));
+        try {
+            execute(dao -> dao.writeBatchOfRounds(namespace, useCase, rounds));
+        } catch (Exception ex) {
+            log.error(
+                    "Failed to write round batch of rounds",
+                    SafeArg.of("namespace", namespace),
+                    SafeArg.of("useCase", useCase),
+                    SafeArg.of(
+                            "seqs",
+                            Streams.stream(rounds).map(PaxosRound::sequence).collect(Collectors.toList())),
+                    ex);
+            throw ex;
+        }
```

Comment:
nit: just warn on these? These shouldn't normally be indicative that timelock is dying